DiceKitTests/FrequencyDistribution_Tests.swift

```diff
@@ -0,0 +1,246 @@
+//
+//  FrequencyDistribution_Tests.swift
+//  DiceKit
+//
+//  Created by Brentley Jones on 7/25/15.
+//  Copyright © 2015 Brentley Jones. All rights reserved.
+//
+
+import XCTest
+import Nimble
+
+import DiceKit
+
+/// Tests the `FrequencyDistribution` type
+class FrequencyDistribution_Tests: XCTestCase {
+
+}
+
+// MARK: - init() tests
+extension FrequencyDistribution_Tests {
+    
+    func test_init_shouldSucceedWithOutcomesPerValue() {
+        // TODO: SwiftCheck
+        let outcomesPerValue: FrequencyDistribution.OutcomesPerValue = [1:1, 2:1, 3:4, 4:1]
+        
+        let freqDist = FrequencyDistribution(outcomesPerValue)
+        
+        expect(freqDist.outcomesPerValue) == outcomesPerValue
+    }
+    
+}
+
+// MARK: - Equatable
+extension FrequencyDistribution_Tests {
+    
+    // TODO: Equatable
+    
+}
+
+// MARK: - Primitive Operations
+extension FrequencyDistribution_Tests {
+    
+    func test_subscript() {
+        // TODO: SwiftCheck
+        let outcomesPerValue: FrequencyDistribution.OutcomesPerValue = [1:1.0, 2:1.0, 3:4.0, 4:1.0]
+        let value = 3
+        let expectedOutcome = outcomesPerValue[value]
+        let freqDist = FrequencyDistribution(outcomesPerValue)
+        
+        let outcome = freqDist[value]
+        
+        expect(outcome) == expectedOutcome
+    }
+    
+    func test_approximatelyEqual_shouldNotEqualForDifferentNumberOfValues() {
+        let delta = 1e-16 // 64-bit. What about 32-bit?
```

Comment:
Should we make a TODO to make this a global somewhere? I think I've seen this multiple times now.